test.js

```diff
@@ -7,15 +7,15 @@ const newsSubPathRE = /^\/news\//;
 redirects.forEach((obj, i) => {
   const from = obj.from;
   const to = obj.to;
-  const mediumRedirect = to.includes("https://medium.com/");
+  const externalUrl = to.includes("https://");
```

Comment:
```suggestion
  const externalUrl = to.startsWith("https://");
```
Nitpick, but in principle someone could put "https://" as part of a link.

Reply:
@ojeytonwilliams, thanks for catching this. You're right, there's no need for `fs` or `JSON.parse` here. Think I was just more familiar with loading/working with JSON files that way when I first wrote this.